laser_slam/src/benchmarker.cpp

```diff
@@ -5,109 +5,219 @@
 #include <iomanip>
 #include <iostream>
 
+#include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
```

Comment:
We managed not to use boost so far. Would it be hard work to replace it with standard c++ functions?